### DISCUSSION TITLE
sensore qnap

### DISCUSSION THREAD

**Discussion by dibummer** [indomus/forum#1913] [GENERALE - Discussioni generiche]
un saluto a tutti,
qualcuno potrebbe indicarmi come monitorare i sensori di qnap nelle sole ore in cui è acceso il Nas?
grazie

**Comment by snakuzzo** [comment]
Che senso avrebbe questa cosa ?
La dico meglio...qual è il tuo scopo ?
Ma poi...di che sensori parliamo ? Cosa c'è che attualmente non ti funziona ?

Per ogni casistica penso tu possa risolvere a prescindere con i template sensor

**Comment by dibummer** [comment]
Siccome il qnap è acceso dalle ore 06:00 alle ore 22:00 e nelle restante ore in cui è spento mi arrivano numerosi messaggi di errori nel registro, i quali mi dicono che l’host non è raggiungibile.
Per questo
Grazie



Questo messaggio e i suoi allegati sono indirizzati esclusivamente alle persone indicate. La diffusione, copia o qualsiasi altra azione derivante dalla conoscenza di queste informazioni sono rigorosamente vietate. Qualora abbiate ricevuto questo documento per errore siete cortesemente pregati di darne immediata comunicazione al mittente e di provvedere alla sua distruzione, Grazie.

This e-mail and any attachments is confidential and may contain privileged information intended for the addressee(s) only. Dissemination, copying, printing or use by anybody else is unauthorised. If you are not the intended recipient, please delete this message and any attachments and advise the sender by return e-mail, Thanks.

Leonardo Di Bello
***@***.***

**Reply by dibummer** [reply]
funziona tutto e solo per non ricevere quelle segnalazioni di errori.

**Reply by snakuzzo** [reply]
immagino tu stia usando l'integrazione nativa QNAP.
Anche se usassi soluzioni alternative, ci sarebbe sempre l'errore (tra le altre cose il NAS solitamente si accende e si lascia lì, senza spegnerlo).
Ad ogni modo, al posto tuo me ne fregherei, ma se proprio non vuoi vedere quella roba nei log, puoi usare [logger](https://www.home-assistant.io/integrations/logger/) per non far scrivere nulla.

Dovrebbe bastarti qualcosa tipo questo...ma è da testare (magari sostituisci la riga relativa all'integrazione con quello che vedi nel messaggio di errore)

```yaml
logger:
  default: warning
  logs:
    homeassistant.components.qnap: notset
```

Così non dovresti vedere nulla relativo a quell'integrazione (che può avere i suoi pro e i suoi contro).
In alternativa potresti anche creare un'automazione che ad una certa ora attiva il log completo e ad un'altra lo disattiva...sfruttando sempre logger e [servizi messi a disposizione](https://www.home-assistant.io/integrations/logger/#services)

**Reply by dibummer** [reply]
grazie.
